.circleci/config.yml

```diff
@@ -192,7 +192,23 @@ aliases:
       name: Install coureutils
       command: |
         brew install coreutils
-
+  - &generate_appcenter_instabug_sourcemap_android_and_ios
+    run:
+      name: Generate and upload sourcemap for Android and iOS on AppCenter
+      command: |
+        if [ "$CIRCLE_BRANCH" = "develop" ]; then
+          env
+          echo "Version is: " $CIRCLE_BUILD_NUM
+          yarn android:sourcemap & yarn ios:sourcemap
+          curl -X POST 'https://api.instabug.com/api/sdk/v3/symbols_files' -F "app_version={\"code\": ${CIRCLE_BUILD_NUM} ,\"name\": ${CIRCLE_BUILD_NUM} }" -F "symbols_file=@/home/circleci/pillarwallet/android-sourcemap.json" -F "application_token=${INSTABUG_APPCENTER_TOKEN}" -F "platform=react_native" -F "os=android" && curl -X POST 'https://api.instabug.com/api/sdk/v3/symbols_files' -F "app_version={\"code\": ${CIRCLE_BUILD_NUM} ,\"name\": ${CIRCLE_BUILD_NUM} }" -F "symbols_file=@/home/circleci/pillarwallet/ios-sourcemap.json" -F "application_token=${INSTABUG_APPCENTER_TOKEN}" -F "platform=react_native" -F "os=ios"
+        elif [ "$CIRCLE_BRANCH" = "master" ]; then
+          env
+          echo "Version is: " $CIRCLE_BUILD_NUM
+          yarn android:sourcemap & yarn ios:sourcemap
+          curl -X POST 'https://api.instabug.com/api/sdk/v3/symbols_files' -F "app_version={\"code\": ${CIRCLE_BUILD_NUM} ,\"name\": ${CIRCLE_BUILD_NUM} }" -F "symbols_file=@/home/circleci/pillarwallet/android-sourcemap.json" -F "application_token=${INSTABUG_PROD_TOKEN}" -F "platform=react_native" -F "os=android" && curl -X POST 'https://api.instabug.com/api/sdk/v3/symbols_files' -F "app_version={\"code\": ${CIRCLE_BUILD_NUM} ,\"name\": ${CIRCLE_BUILD_NUM} }" -F "symbols_file=@/home/circleci/pillarwallet/ios-sourcemap.json" -F "application_token=${INSTABUG_PROD_TOKEN}" -F "platform=react_native" -F "os=ios"
```

Comment:
are we using the CircleCI build num version for prod as well? I presume answer is yes since this is executed way before we have a new tag and prod build 😄